R/rnet_functions.R

```diff
@@ -0,0 +1,131 @@
+# Wrapper function for making rnets
+# r routes
+# type character e.g. "school_fastest"
+# ncores passed to overline
+# regionalise passed to overline
+
+#TODO: inconsistent capitalisation Quietness vs quietness
+
+make_rnets = function(r, ncores = 20, regionalise = 1e5){
+  r = r[r$bicycle_go_dutch > 0 | r$bicycle > 0,]
+  r$Gradient = round(r$gradient_smooth * 100)
+  r$Quietness = round(r$quietness)
+  
+  rnet = stplanr::overline2(r, 
+                            attrib = c("bicycle","bicycle_go_dutch","Quietness","Gradient"), 
+                            fun = list(sum = sum, max = first),
+                            ncores = ncores, regionalise = regionalise)
+  
+  rnet = rnet[,c("bicycle_sum","bicycle_go_dutch_sum","Gradient_max","Quietness_max")]
+  names(rnet) = gsub("_sum$","",names(rnet))
+  names(rnet) = gsub("_max$","",names(rnet))
+  
+  # Suppress low values, replace with 3
+  rnet$bicycle <- round_sdc(rnet$bicycle)
+  rnet$bicycle_go_dutch <- round_sdc(rnet$bicycle_go_dutch)
+  
+  rnet
+}
+
+# Function to merge list of rnets into a single rnet
+# rnl list of rnets
+
+rnl = rnet_commute_list
```

Comment:
@mem48 this was why you got the "rnet_commute_list not found" error. This is a typo and should be fixed before this is merged.